app/views/case_court_reports/index.html.erb

```diff
@@ -47,6 +47,9 @@
   }
 
   const setTimeZone = () => {
+     if(!$('#case-selection').val()){
+      alert('Select an active case and specify the date range.');
+    }
```

Comment:
honestly i don't love how the default alert looks, I would like it to highlight the field that has the error.

I would be fine with this if I can be done as an HTML attribute otherwise lets just ditch the client side JS requirement.